Add Splitter (#12)
makerdao/dss-flappers: README.md

# Dss Flappers

> Implementations of MakerDao surplus auctions, triggered on `vow.flap`.

Reviewer: Need to update the documentation about `FlapperMom` (replace with `SplitterMom` etc..).

> ### Splitter

Reviewer: "the `vow` is expected to hold at least the swapped amount" =>
"the `vow` is expected to hold at least the consumed amount"

# Dss Flappers

> Implementations of MakerDao surplus auctions, triggered on `vow.flap`.

Reviewer 2: ```suggestion
Implementations of MakerDAO surplus auctions, triggered on `vow.flap`.
```

Allows for scaling down an oracle price by a certain value. This can be useful when the `gem` is a redenominated version of an existing token, which already has a reliable oracle.

### Splitter

> Exposes a `kick` operation to be triggered periodically. Its logic withdraws `DAI` from the `vow` and splits it in two parts. The first part (`burn`) is sent to the underlying `flapper` contract to be burned. The second part (`WAD - burn`) is distributed as reward to a `farm` contract. The `kick` cadence is determined by `flapper.hop()`.

Reviewer: ```suggestion
Exposes a `kick` operation to be triggered periodically. Its logic withdraws `DAI` from the `vow` and splits it in two parts. The first part (`burn`) is sent to the underlying `flapper` contract to be processed by the burn engine. The second part (`WAD - burn`) is distributed as reward to a `farm` contract. The `kick` cadence is determined by `flapper.hop()`.
```

Configurable Parameters:
* `flapper` - The underlying burner strategy (e.g. the address of `FlapperUniV2SwapOnly`).

> * `burn` - The percentage of the `vow.bump` to be moved to the underlying `flapper`. For example, a value of 0.70 \* `WAD` corresponds to a 70% burn.

Reviewer: ```suggestion
* `burn` - The percentage of the `vow.bump` to be moved to the underlying `flapper`. For example, a value of 0.70 \* `WAD` corresponds to funneling 70% of the `DAI` to the burn engine.
```
